sdks/python/apache_beam/ml/inference/huggingface_inference_it_test.py

```diff
@@ -74,6 +75,37 @@ def test_hf_language_modeling(self):
       predicted_predicted_text = predictions_dict[text]
       self.assertEqual(actual_predicted_text, predicted_predicted_text)
 
+  def test_hf_pipeline(self):
+    test_pipeline = TestPipeline(is_integration_test=True)
+    # Path to text file containing some questions and context
+    input_file = 'gs://apache-beam-ml/datasets/custom/questions.txt'
+    output_file_dir = 'gs://apache-beam-ml/hf/testing/predictions'
```

Comment:
Will this break if the underlying model changes? Could we pin to a single revision?

Reply:
Ooh, actually same question about the existing tests

Reply:
Sounds good. yeah internal changes would corrupt our model. Pinning it to `4.30.0` for `huggingface_tests_requirements.txt`.

Reply:
I actually mean the model itself that the pipeline is built on top of, e.g. if they update from v2 to v3 - https://huggingface.co/deepset/roberta-base-squad2/commit/deedc3e42208524e0df3d9149d1f26aa6934f05f

I guess that's _that_ likely to happen, but we could just pin to v2 or even a commit sha with the revision parameter - https://huggingface.co/docs/transformers/main_classes/pipelines#transformers.pipeline.revision

Reply:
Got it, updated the test, thanks!